### PR TITLE
Make feedback links configurable

### PR DESCRIPTION
**Pull request by Cruikshanks**
Part of work for WAS-180 Set up feedback link to GDS

At the end of journey and in the beta banner we provide links to users to give feedback about the service. We want these to be configurable by environment so that we can go to a stub in non-production environments, and the real thing in production.

This is part of that change and covers creating the config values for use in the views, which can then be set by initilizers in the user facing projects.

### CODE REVIEW

File: app/views/shared/_phase_banner.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="phase-banner">
   <p>
     <strong class="phase-tag">BETA</strong>
-    <span><%= t(".heading_html", :href => link_to(t(".feedback"), "#")) %></span>
+    <span><%= t(".heading_html", :href => link_to(t(".feedback"), WasteExemptionsShared.config.complete_page_feedback_uri)) %></span>
```
Comment (rickclare) [inline app/views/shared/_phase_banner.html.erb]: Shouldn't this be pointing to `banner_feedback_uri` that you've configured in _configuration.rb_?

**Comment by timstone** [comment]
QA passed over to you @rickclare to PR please

File: app/views/shared/_phase_banner.html.erb (hunk 1 above)
Comment (Cruikshanks) [inline app/views/shared/_phase_banner.html.erb]: Monkey nuts! Good spot! Will resolve.

**Comment by rickclare** [comment]
Thanks @Cruikshanks Shall I rebase and merge this into develop now?

**Comment by Cruikshanks** [comment]
Back to you @rickclare I had to merge in the end as couldn't get rebasing to get past the point the overcommit change came in (kept dropping out into a detached head). However it should be good to go now.

Only comment I would make on my own code is that it is missing a test to confirm that the link is **x** in production and **y** every else but not sure how I would tackle this and don't want to take up precious dev time from you guys trying to figure it out.

If you think its easy and can drop me a few keywords to google I'll give it a go, else merge away and lets put this puppy to bed!

**Pull request merged**